avogadro/qtplugins/selectiontool/selectiontool.cpp

```diff
@@ -39,6 +39,10 @@
 #include <QtGui/QMouseEvent>
 #include <QtWidgets/QAction>
 
+#include <iostream>
```

Comment:
Did we need this? If you want to debug in any of the Qt classes:
```
#include <QDebug>

..
qDebug() << " debugging message ";
```

Reply:
Ups sorry.